packages/mediacenter/kodi-binary-addons/screensaver.stars/package.mk

```diff
@@ -3,9 +3,9 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="screensaver.stars"
-PKG_VERSION="2.1.2-Leia"
-PKG_SHA256="8873de9be41365af9dc639e3624d887f7b18d8e1f43e9a69af0c3154f9b59020"
-PKG_REV="2"
+PKG_VERSION="2.1.3-Leia"
```

Comment:
there is a matrix branch (+ release)

Reply:
Kodi is still using the Leia branch for this repo: https://github.com/xbmc/repo-binary-addons/blob/Matrix/screensaver.stars/screensaver.stars.txt